ddl/table.go

```diff
@@ -857,12 +857,14 @@ func (w *worker) onShardRowID(d *ddlCtx, t *meta.Meta, job *model.Job) (ver int6
 }
 
 func verifyNoOverflowShardBits(s *sessionPool, tbl table.Table, shardRowIDBits uint64) error {
+	if shardRowIDBits == 0 {
```

Comment:
Can you add some tests for it?

Reply:
It is not convenient to construct such a test. Please check the "manual test" in the PR description.